afs/media/js/views/components/reports/sampling-activity.js

```diff
@@ -0,0 +1,115 @@
+define([
+    'jquery',
+    'underscore',
+    'knockout',
+    'arches',
+    'utils/resource',
+    'utils/report',
+    'views/components/reports/scenes/name'
+], function($, _, ko, arches, resourceUtils, reportUtils) {
+    return ko.components.register('sampling-activity-report', {
+        viewModel: function(params) {
+            var self = this;
+            params.configKeys = ['tabs', 'activeTabIndex'];
+            Object.assign(self, reportUtils);
+            self.sections = [
+                {'id': 'name', 'title': 'Names and Classifications'}, 
+            ];
+            self.reportMetadata = ko.observable(params.report?.report_json);
+            self.resource = ko.observable(self.reportMetadata()?.resource);
+            self.displayname = ko.observable(ko.unwrap(self.reportMetadata)?.displayname);
+            self.activeSection = ko.observable('name');
+            self.dataConfig = {
+                exactMatch: undefined
+            };
+            self.nameCards = {};
+
+            if(params.report.cards){
+                const cards = params.report.cards;
+                
+                self.cards = self.createCardDictionary(cards)
+
+                self.nameCards = {
+                    name: self.cards.name,
+                    identifier: self.cards.Identifier,
+                    exactMatch: self.cards.ExactMatch,
+                    type: self.cards.Classification
+                }
+            }
+
+            if (params.summary) {
+
+                this.editorLink = arches.urls.resource_editor + this.report.attributes.resourceid;
```

Comment:
That is correct. The sampling and provenance activity did not have summary report to begin with. So, it was copied from textual work and left it there, hoping it can be used as a template. But if it is confusing, they should be removed probably both from the viewmodel and the template.

Reply:
They are removed